README.md

```diff
@@ -89,7 +89,8 @@ Here you can see an example on how the configuration file should look like if yo
      "auth": {
          "type": "api_key",
          "api_key": "jfeRFsdfRfSasgsDoNOtTrYToUsErRQwqqdZfghasd"
-     }
+     },
+    "headers":{"X-OpenAI-Api-Key":"OPEN_AI_KEY"}
```

Comment:
Maybe defined some other API key headers for Cohere and some other, to leave clear that you can pass multiple vectorizer headers